src/cascadia/TerminalConnection/AzureConnection.idl

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.Terminal.TerminalConnection
 {
     [default_interface] runtimeclass AzureConnection : ITerminalConnection
     {
-        AzureConnection(UInt32 rows, UInt32 columns);
+        static Boolean IsAzureConnectionAvailable();
+		AzureConnection(UInt32 rows, UInt32 columns);

```

Comment:
nit: these guys aren't aligned

Reply:
Idk why this was happening, it looked fine when I opened it up in visual studio... I fixed it so it looks weird on visual studio for me but should be fine on github